compiler/compile/OMRCompilation.cpp

```diff
@@ -2499,8 +2499,7 @@ void OMR::Compilation::diagnosticImplVA(const char *s, va_list ap)
       va_list copy;
       va_copy(copy, ap);
       char buffer[256];
-      TR::IO::vfprintf(self()->getOutFile(), self()->getDebug()->getDiagnosticFormat(s, buffer, sizeof(buffer)/sizeof(char)),
-                 copy);
+      TR::IO::vfprintf(self()->getOutFile(), s, copy);
```

Comment:
Why did we ever need `copy`? Or this method for that matter; why couldn't any of this just be done in `OMR::Compilation::diagnosticImpl(const char *s, ...)`? Not sure if cleaning this up is part of a future PR in this series of PRs.

Reply:
Fixed in forced push.